Modix.Services/BehaviorBase.cs

```diff
@@ -27,14 +29,16 @@ public BehaviorBase(IServiceProvider serviceProvider)
             => Dispose(false);
 
         /// <inheritdoc />
-        public async Task StartAsync()
+        public async Task StartAsync(
+            CancellationToken cancellationToken)
         {
             await OnStartingAsync();
```

Comment:
The cancellation token here should be passed on

Reply:
Agreed. `BehaviorBase` is part of the prior implementation of `IBehavior` and the only reason I didn't remove it outright is because the process of migrating features that use it is non-trivial. It involves replacing `.SelfExecuteAsync()` with `IMessageDispatcher.Dispatch()`.

I'll add a `[Deprecated]` tag to this instead.